portal/src/graphql/portal/OnboardingConfigAppScreen.tsx

```diff
@@ -0,0 +1,300 @@
+import React, { useCallback, useContext, useMemo } from "react";
+import { Navigate, useParams } from "react-router-dom";
+import produce from "immer";
+import { FormattedMessage } from "@oursky/react-messageformat";
+import { Checkbox, FontIcon, Label, Text } from "@fluentui/react";
+
+import {
+  IdentityType,
+  LoginIDKeyType,
+  PortalAPIAppConfig,
+  SecondaryAuthenticationMode,
+  SecondaryAuthenticatorType,
+  VerificationClaimsConfig,
+} from "../../types";
+import ShowError from "../../ShowError";
+import ShowLoading from "../../ShowLoading";
+import ScreenHeader from "../../ScreenHeader";
+import {
+  AppConfigFormModel,
+  useAppConfigForm,
+} from "../../hook/useAppConfigForm";
+import { useAppConfigQuery } from "./query/appConfigQuery";
+import OnboardingFormContainer from "./OnboardingFormContainer";
+import styles from "./OnboardingConfigAppScreen.module.scss";
+
+interface PendingFormState {
+  identities: Set<IdentityType>;
+  loginIDKeys: Set<LoginIDKeyType>;
+}
+
+interface FormState {
+  pendingForm: PendingFormState;
+}
+
+function constructFormState(_config: PortalAPIAppConfig): FormState {
+  return {
+    pendingForm: {
+      identities: new Set<IdentityType>(),
+      loginIDKeys: new Set<LoginIDKeyType>(),
```

Comment:
By default, we should enable Login ID identity and the Email Login ID.

Reply:
Found that we cannot simply change the initial state to default value. If we change the initial state to default value, the AppConfigFormModel will treat the form isDirty as false and not allow saving.  Let me think how to handle it..

Reply:
And we should disable save button if nothing has been selected for the first question.

Reply:
Updated in 90d379c